requirements.txt

```diff
@@ -1,16 +1,21 @@
 cogment[generate]==2.2.2
 grpcio==1.43.0
-hydra-core==1.3.2
-mlflow==1.26.1
+hydra-core==1.2.0
+mlflow==1.30.0
 names_generator~=0.1.0
 opencv-python~=4.5.5
 PyYAML~=5.3.1
 starlette==0.20.0
 uvicorn==0.17.6
 
 # environments
-gym[atari,box2d,classic_control,mujoco]>=0.25,<0.26
-PettingZoo[classic]>=1.18,<1.19
+gym[box2d,classic_control, mujoco]>=0.25,<0.26
+PettingZoo[atari, classic]~=1.22.2
+SuperSuit~=3.7.0
+tinyscaler~=1.2.4
+Gymnasium~=0.26.3
```

Comment:
can't we just get rid of gym ?

Reply:
?

Reply:
@wduguay-air has already created [a separate issue](https://github.com/cogment/cogment-verse/issues/134) for this